spacy_llm/tasks/textcat.py

```diff
@@ -137,6 +137,7 @@ def _format_response(self, response: str) -> Dict[str, float]:
         The returned dictionary contains the labels mapped to their score.
         """
         categories: Dict[str, float]
+        response = response.strip()
```

Comment:
I'm ok with this solution! Thank you!

Optional suggestion: I'm wondering if there's a way we can reuse the normalizer function (that also does the stripping) here? 

Let's also add a test case in "test_binary_labels_are_correct"? Or a new test in case it doesn't fit!

Reply:
you're right we should use the normalizer for sure for this. Made an update, let me know what you think to my question below.

Reply:
IIRC, the normalizer is mainly meant to compare different labels, right? This is about stripping whitespace from the response, which feels like a natural thing to always do?

Reply:
@svlandeg If you put it that way I think that makes sense. @kabirkhan I'm ok reverting the commits from fc60358! 🙇